daml-script/runner/src/main/scala/com/digitalasset/daml/lf/engine/script/LedgerInteraction.scala

```diff
@@ -434,6 +433,10 @@ class IdeClient(val compiledPackages: CompiledPackages) extends ScriptLedgerClie
                     // Capture the result and exit.
                     result = Success(Right(results.toSeq))
                 }
+              case Right(Left(x)) =>
+                result = Failure(new RuntimeException(s"Unexpected abort: $x"))
```

Comment:
Any chance we can do better than a `RuntimeException`?

Reply:
I will let that for `daml-application-runtime` team.

Reply:
Fair point, I need to brush up my knowledge of sensible exception types in Scala.